.github/workflows/build-and-test.yml

```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
 
 env:
-  GO_VERSION: 1.19
+  GO_VERSION: 1.20
```

Comment:
Put quotes around the `"1.20"`. I think it is interpreting it incorrectly and trying to use Go 1.2 instead.

Reply:
Yes!!! That fixed the issue. Thank you for your help!